pom.xml

```diff
@@ -20,30 +20,82 @@
 
     <!-- Deployment Details -->
     <repositories>
-        <repository>
-            <name>Spicule-Kythera GitLab</name>
-            <id>gitlab-maven</id>
-            <url>${env.CI_SERVER_URL}/api/v4/projects/${env.CI_PROJECT_ID}/packages/maven</url>
-            <snapshots>
-                <enabled>true</enabled>
-            </snapshots>
-        </repository>
+        <!-- AShot Repo -->
         <repository>
             <id>gitlab-maven-ashot</id>
             <url>https://gitlab.com/api/v4/projects/28250236/packages/maven</url>
         </repository>
-    </repositories>
 
-    <distributionManagement>
+        <!-- Clojars Repo -->
         <repository>
```

Comment:
I think this and the bit above still needs reverting cause the CI job is failing.